ignite/engine/events.py

```diff
@@ -231,7 +273,7 @@ def print_epoch(engine):
         # print_epoch handler is now unregistered
     """
 
-    def __init__(self, event_name: Union[EventWithFilter, CallableEvents, Enum], handler: Callable, engine):
+    def __init__(self, event_name: Union[CallableEventWithFilter, Enum], handler: Callable, engine):
```

Comment:
Currently, I would say `event_name: Any`. We need to decide about it and fix all typings according to it.